packages/fxa-shared/index.ts

```diff
@@ -54,6 +54,7 @@ module.exports = {
   dto: {
     auth: {
       payments: {
+        iapSubscription,
```

Comment:
@bbangert : Looking at your [PR](https://github.com/mozilla/fxa/pull/11213) where you added the `invoice` DTOs and the /invoice/preview route, I noticed you added the DTO to `fxa-shared/index.ts. What is this for? I've been able to use them by directly importing them e.g. in the auth server.

Reply:
It looks like this export is used in some places, so I'm going to leave it (e.g. `popular-email-domains.js`).